docs/spec.md

```diff
@@ -4145,17 +4145,17 @@ Afterwards, within each `process_group`:
 
 #### Constraints
 
-* (C1) `dim(operand, scatter_dimension) % dim(process_groups, 1) = 0`.
```

Comment:
C8 doesn't say that scatter_dimension size is divisible by the subgroup size. We need a separate constraint, e.g. C1, to enforce that.

Reply:
You're right. However, note that verification for C1 is infeasible since we do not know the shape of `process_groups` statically.